Dockerfile

```
@@ -1,24 +1,36 @@
+
+COPY . .
+RUN yarn build
+
+# --- runtime image ---
 FROM node:22-alpine
 
 WORKDIR /app
 
-# Install yt-dlp and its dependencies (always latest from pip)
-# Also install build tools needed for native Node.js modules
 RUN apk add --no-cache python3 py3-pip ffmpeg build-base autoconf automake libtool g++ \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)